testcases/open_posix_testsuite/conformance/interfaces/mq_timedreceive/5-1.c

```diff
@@ -29,6 +29,10 @@
 #include <time.h>
 #include "posixtest.h"
 
+#ifndef TIME_T_MAX
+#	define TIME_T_MAX	(time_t)((1UL << ((sizeof(time_t) << 3) - 1)) - 1)
+#endif
+
```

Comment:
Can we please put this fallback definition into a header, maybe even to a file in open_posix_testsuite/include/?

Reply:
I thought the same when copying it the third time, just didn't know where to put it. Is open_posix_testsuite/include/timespec.h ok for you?

Reply:
Sounds good.